websites/A/Aniorb/presence.ts

```diff
@@ -0,0 +1,107 @@
+const presence = new Presence({
+  clientId: "776312991939428373"
+}),
+strings = presence.getStrings({
+  play: "presence.playback.playing",
+  pause: "presence.playback.paused",
+  browsing: "presence.activity.browsing"
+});
+let video = {
+video: false,
+duration: 0,
+currentTime: 0,
+paused: true
+};
+  const timestamps = presence.getTimestamps(
+    Math.floor(video.currentTime),
+    Math.floor(video.duration)
+  );
+presence.on(
+"iFrameData",
+(data: {
+  video: boolean;
+  duration: number;
+  currentTime: number;
+  paused: boolean;
+}) => {
+  video = data;
+}
+);
+presence.on("UpdateData", async () => {
+const presenceData: PresenceData = {
+  largeImageKey: "aniorb"
+};
+   if (document.location.pathname.includes("/genre")) {
+  presenceData.details = "Browsing Through";
+  presenceData.state = "The Genres";
+  presenceData.smallImageKey = "reading";
+} else if (document.location.pathname.includes("/search")) {
+  presenceData.details = "Searching for some anime...";
+  presenceData.smallImageKey = "search";
+} else if (document.location.pathname.includes("/popular")) {
+  presenceData.details = "Browsing Through";
+  presenceData.state = "Popular Anime";
+  presenceData.smallImageKey = "popular";
+} else if (document.location.pathname.includes("/recentlyadded")) {
+  presenceData.details = "Browsing through";
+  presenceData.state = "Recently Added";
+  presenceData.smallImageKey = "recent";
+} else if (document.location.pathname.includes("/myList")) {
+  presenceData.details = "Browsing through";
+  presenceData.state = "My List";
+  presenceData.smallImageKey = "heart";
+} else if (document.location.pathname.includes("/details")) {
+  presenceData.details = "Browsing through";
+  presenceData.state = "Anime Details";
+  presenceData.smallImageKey = "reading";
+}
+else if (document.querySelector("#__next > div > div.flex.justify-between > div.w-full.justify-center.items-center.min-h-screen.lg\\:h-full.lg\\:w-10\\/12 > div > div.flex.flex-col.pb-2.xl\\:w-player.justify-between.items-center.w-full.text-white.my-4 > div.w-full.py-4.uppercase.flex.flex-col.items-start.lg\\:items-start > span").textContent.toUpperCase() !== null) {

```

Comment:
Declare this query somewhere maybe or just make it shorter pretty sure you can do that